css/main.scss

```diff
@@ -99,7 +101,7 @@ a {
 }
 p.divider {
 	&:before {
-		content:""; display:block; width:48px; border-bottom:1px solid var(--subtle); margin-bottom:20px;
+		content:""; display:block; width:48px; border-bottom:1px solid var(--link_subtle); margin-bottom:20px;
```

Comment:
hr to match link color

Reply:
<img width="746" alt="hr_dark" src="https://user-images.githubusercontent.com/1668411/84416270-97b9a380-abd9-11ea-92fb-5e204a4bb10b.png">
<img width="746" alt="hr_light" src="https://user-images.githubusercontent.com/1668411/84416279-98523a00-abd9-11ea-9ff1-6c299b6ff886.png">